Project Management/Schedule/Zowe PI & Sprint Cadence.md

```diff
@@ -1,123 +1,119 @@
+
+#### 3.3.0 
+- Code Freeze: July 7, 2025
+- RC Build: July 8, 2025
+- Testing: July 8, 2025 - August 3, 2025
+- GA: August 4, 2025
+- System Demo: August 12, 2025
+
+#### 3.4.0 
+- Code Freeze: October 6, 2025
+- RC Build: October 7, 2025
+- Testing: October 7, 2025 - November 2, 2025
+- GA: November 3, 2025
+- System Demo: November 11, 2025
+
+#### 3.5.0 
+- Code Freeze: January 5, 2026
+- RC Build: January 6, 2026
+- Testing: January 6, 2026 - February 1, 2026
+- GA: February 2, 2026
+- System Demo: February 10, 2026
+
+#### 3.6.0 
+- Code Freeze: April 6, 2026
+- RC Build: April 7, 2026
+- Testing: April 7, 2026 - May 3, 2026
+- GA: May 4, 2026
+- System Demo: May 12, 2026
+
+#### 3.7.0 
+- Code Freeze: July 6, 2026
+- RC Build: July 7, 2026
+- Testing: July 7, 2026 - August 2, 2026
+- GA: August 3, 2026
+- System Demo: August 11, 2026
 
 ### V2
 
-#### 2.15 </br>
-- Code Freeze 2024/02/20<br>
-- RC Build 2024/02/21<br>
-- System Demo 2024/03/11<br>
-- Testing 2024/02/21 - 2024/03/03<br>
-- GA 2024/03/04<br>
-
-#### 2.16 </br>
-- Code Freeze 2024/04/30<br>
-- RC Build 2024/05/01<br>
-- System Demo 2024/05/20<br>
-- Testing 2024/05/01 - 2024/05/13<br>
-- GA 2024/05/13<br>
-
-#### 2.17 </br>
-- Code Freeze 2024/07/02<br>
-- RC Build 2024/07/03<br>
-- System Demo 2024/07/23<br>
-- Testing 2024/07/03 - 2024/07/14<br>
-- GA 2024/07/15<br>
-
 #### 2.18 </br>
-- Code Freeze 2024/08/13<br>
-- RC Build 2024/08/14<br>
-- System Demo 2024/09/03<br>
-- Testing 2024/08/14 - 2024/08/25<br>
-- GA 2024/08/26<br>
+- Code Freeze: August 13, 2024
+- RC Build: August 14, 2024
+- Testing: August 14, 2024 - August 25, 2024
+- GA: August 26, 2024
+- System Demo: September 03, 2024
 
 #### 2.18.1
-- Code Freeze 2025/01/02<br>
-- RC Build 2025/01/03<br>
-- Testing 2025/01/03 - 2025/01/14<br>
-- GA 2025/01/15<br>
+- Code Freeze: February 10, 2025
+- RC Build: February 11, 2025
+- Testing: February 11, 2025 - February 23, 2025
+- GA: February 24, 2025
+
+### 2.18.2
+- Code Freeze: August 15, 2025
+- RC Build: August 15, 2025
+- Testing: August 16, 2025 - August 28, 2025
+- GA: August 29, 2025
+
+### 2.18.3
+- Code Freeze: February 09, 2026
+- RC Build: February 10, 2026
+- Testing: February 10, 2026 - February 22, 2026
+- GA: February 23, 2026
+
+### 2.18.4
+- Code Freeze: August 17, 2026
+- RC Build: August 18, 2026
+- Testing: August 18, 2026 - August 30, 2026
+- GA: August 31, 2026
+
+### 2.18.5
+- Code Freeze: February 08, 2027
+- RC Build: February 09, 2027
+- Testing: February 09, 2027 - February 21, 2027
+- GA: February 22, 2027
 
 ### V1
 
-#### 1.28.6
-- Code Freeze 2024/03/27<br>
-- RC Build 2024/03/28<br>
-- Testing 2024/03/28 - 2024/04/09<br>
-- GA 2024/04/10<br>
-
-#### 1.28.7
-- Code Freeze 2024/05/14<br>
-- RC Build 2024/05/15<br>
-- Testing 2024/05/15 - 2024/05/26<br>
-- GA 2024/05/27<br>
+#### 1.28.8
+- Code Freeze: October 1, 2024
+- RC Build: October 2, 2024
+- Testing: October 2, 2024 - October 14, 2024
+- GA: October 14, 2024
```

Comment:
Final V1 release 👀

Reply:
Yes, that's the last one, nothing more after this one.